index.bs

```diff
@@ -390,6 +390,74 @@ by a custom configuration.
 {}
    ```
 
+# Security Considerations {#security-considerations}
+
+The Sanitizer API is intended to prevent DOM-Based Cross-Site Scripting
+by traversing a DocumentFragment and removing elements and attributes
```

Comment:
Nit: Perhaps focus on a conversion of a _string_ to a DocumentFragment, instead of traversing an existing DocumentFragment. The first one is what may prevent DOM XSS (as the input string is user controlled). The creation of an inert DocumentFragment and traversing it to remove nodes is an implementation detail.

Reply:
Good point. Thanks!